read-yaml/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Read URL YAML
+description: Reads in YAML file from URL.
+author: Anaconda Inc.
+inputs:
+  path:
+    description: Path to YAML file in this repo to read.
+    required: true
+  scopes:
+    description: Path of keys to the value to extract from file (e.g. foo.bar).
+    required: false
+# cannot set outputs since they are dynamic based on the scopes value
+# outputs:
+#   data:
+#     description: Data read from YAML file.
+#     value: ${{ steps.read_yaml.outputs.data }}
```

Comment:
Is this going to be a problem if the outputs aren't specified?

Reply:
AFAIK no, the outputs are optional and have no impact on what is actually outputted

see https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#outputs

Reply:
Awesome!